frontend/js/components/HistoryTable/HistoryTable.component.js

```diff
@@ -0,0 +1,70 @@
+// import '../../components/HistoryTable/HistoryTableRow.js'
+import Actions from '../../services/Action.service.js';
+
+class HistoryTable extends HTMLElement {
+
+    
+     constructor() {
+        super();
+        this.sessionsData = Actions.getAllSessions();
+    }
+
+    connectedCallback() {
+        this.render();
+        for(let i=0;i<this.sessionsData.length;i++)
+        {
+          document.getElementById(`${i}`).addEventListener('click',function(){ Actions.resumeSession(i)});
+        }
+    }
+
+    render() {
+     
+        this.innerHTML = `
+        <table class="table session-table" style="width: 95%;">
+        <thead>
+          <tr>
+            <th class='col-2 session-table-th2'>Session Name</th>
+            <th class='col-4 session-table-th2'>Date</th>
+            <th class='col-2 session-table-th2'>Time</th>
+            <th class='col-4 session-table-th2'>Action Item</th>
+          </tr>
+        </thead>
+        <tbody id='session-table-content'>
+          ${ this.sessionsData.length>0 ?
+            (
+              // "<hb-history-table-row></hb-history-table-row>"
+              this.sessionsData.map((item,index)=>{
+                return `<tr class='sessions'>
```

Comment:
spaces in notations,
the HTML string starts on the next line from `

Reply:
done.